spec/chewy/index/actions_spec.rb

```diff
@@ -731,4 +731,51 @@
       end
     end
   end
+
+  describe '._reindex' do
```

Comment:
Same about underline and I would add 2 tests to cover default values for each key.